core/flamingock-test-support/src/test/java/io/flamingock/support/validation/impl/AuditSequenceStrictValidatorTest.java

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright 2025 Flamingock (https://www.flamingock.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.flamingock.support.validation.impl;
+
+import io.flamingock.internal.common.core.audit.AuditEntry;
+import io.flamingock.support.domain.AuditEntryDefinition;
+import io.flamingock.support.validation.error.CountMismatchError;
+import io.flamingock.support.validation.error.FieldMismatchError;
+import io.flamingock.support.validation.error.ValidationResult;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static io.flamingock.internal.common.core.audit.AuditEntry.ExecutionType.EXECUTION;
+import static io.flamingock.internal.common.core.audit.AuditEntry.Status.APPLIED;
+import static io.flamingock.internal.common.core.audit.AuditEntry.Status.FAILED;
+import static io.flamingock.support.domain.AuditEntryDefinition.APPLIED;
+import static org.junit.jupiter.api.Assertions.*;
+
+class AuditSequenceStrictValidatorTest {
+
+    private List<AuditEntry> actualEntries;
+
+    @BeforeEach
```

Comment:
In general use `@DisplayName`, since it's clear to see the title and what we are testing